<?xml version='1.0'?>
<launch>
    <arg name="veh"/>
    <arg name="param_file_name" default="default"/>

    <node pkg="generic_line_detector" exec="generic_line_detector_node" name="generic_line_detector_node" 
        namespace="$(var veh)">
        <param name="veh" value="$(var veh)"/>
        <param name="param_file_path" 
            value="$(find-pkg-share generic_line_detector)/config/$(var param_file_name).yaml"/>
    </node>
</launch>
